src/ray/rpc/node_manager/node_manager_client_pool.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 The Ray Authors.
+// Copyright 2017 The Ray Authors.
```

Comment:
It seems that we need to:

* Ignore the year number in the copyright message when checking.
* Generate the year number of the current year when fixing.

Reply:
@edoakes Hi, do you know the rule of copyright? I found `node_manager_client_pool.h` and  `core_worker_client_pool.h` use '2020', but other files used '2017'.

Reply:
Hmm I'm not actually sure about this... @pcmoritz do you know? I think last time we froze everything at 2017 but not sure why.

Reply:
@pcmoritz can you help to see this?